cmd/fallocate.go

```diff
@@ -20,6 +20,6 @@ package cmd
 
 // Fallocate is not POSIX and not supported under Windows
 // Always return successful
-func Fallocate(fd int, offset int64, len int64) error {
+func Fallocate(fd int, offset int64, length int64) error {
```

Comment:
Same here `len` is synonymous to length, no reason to rename it. The corresponding function in fallocate_linux.go is untouched..

Reply:
It's a builtin function https://golang.org/pkg/builtin/#len